src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java

```diff
@@ -135,6 +136,23 @@
     @Enumerated(EnumType.STRING)
     private DiseaseGeneticModifierRelation diseaseGeneticModifierRelation;
 
+    @FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer")
+    @KeywordField(name = "createdBy_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
+    @JsonView(View.FieldsOnly.class)
+    private String createdBy;
+    
+    @FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer")
+    @KeywordField(name = "modifiedBy_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
+    @JsonView(View.FieldsOnly.class)
+    private String modifiedBy;
+    
+    @Basic
+    @JsonView(View.FieldsOnly.class)
+    private OffsetDateTime creationDate;
+    
+    @Basic
+    @JsonView(View.FieldsOnly.class)
+    private OffsetDateTime dateLastModified;
```

Comment:
@markquintontulloch it might be better to use either a ZonedDateTime or a LocalDateTime for these?

Reply:
I went with Offset as that's the format the DQMs have traditionally submitted in.  We can always change that, or convert on our side, but what's the reason for preferring Zoned over Offset?

Reply:
So just did some reading...

`First, it doesn't make sense (without conversions) to directly compare two dates with full timezone information. Therefore, we should always prefer storing OffsetDateTime in the database over the ZonedDateTime, as dates with a local time offset always represent the same instants in time.`

I didn't know this but looks like we are good to go.